media/js/mozorg/contact-spaces.js

```diff
@@ -115,12 +115,16 @@
             //set the current tab navigation item
             $('ul.category-tabs li[data-id="' + tab + '"]').addClass('current');
 
-            // set the current list menu navigation item
-            $('.nav-category li[data-id="' + id + '"]').addClass('current');
+            if (tab === 'spaces') {
+                // set the current list menu navigation item for the active space
+                $('.nav-category li[data-id="' + id + '"]').addClass('current');
```

Comment:
I believe this line is still needed for both `spaces` and `communities`, as both have side navs.